homeassistant/helpers/aiohttp_client.py

```diff
@@ -319,7 +327,7 @@ def _async_get_connector(
     if verify_ssl:
         ssl_context: SSLContext = ssl_util.get_default_context()
```

Comment:
If verify is true it looks like ciphers are ignored

Reply:
True, that is because I did not know how to make the SSL_CONTEXT for the INSECURE option:
https://github.com/home-assistant/core/blob/f77d0d6ccc90a378921c2540ad854ab6e8296c13/homeassistant/util/ssl.py#L165-L182

Would it just be relplacing `context.set_ciphers(SSL_CIPHER_LISTS[SSLCipherList.INTERMEDIATE]) `?

I am espicially unsure about
```
context.options |= ( 
         ssl.OP_NO_SSLv2 | ssl.OP_NO_SSLv3 | ssl.OP_CIPHER_SERVER_PREFERENCE 
     ) 
```
Should I just leave that the same for the INSECURE option?

Reply:
I just added the option for normal SSL_CONTEXTS, so the chiphers are not ignored anymore when verfity is true.
@bdraco could you check if I did it correctly?